home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1745,7 +1763,7 @@ components:
           minLength: 9
           maxLength: 320
           type: string
-          example: test.receive.messages@gmail.com
+          example: "test.receive.messages@gmail.com"
     UpdatePhoneContact:
```

Comment:
1) Am I correct in understanding that when creating or updating an email contact, you need to enter email twice - one as a property of the base model and the other as a property of the child model?
2) In all the models above, we set the number of characters for e-mail from 5 to 320, and here - from 9 to 320. Is this a typo?

Reply:
> 1. Am I correct in understanding that when creating or updating an email contact, you need to enter email twice - one as a property of the base model and the other as a property of the child model?
> 2. In all the models above, we set the number of characters for e-mail from 5 to 320, and here - from 9 to 320. Is this a typo?

1. It looks like a mistake, give me some time to find out
2. No, it isn't a typo, that's the way it's meant to be

Reply:
> 1. Am I correct in understanding that when creating or updating an email contact, you need to enter email twice - one as a property of the base model and the other as a property of the child model?

After looking at it with a more fresh glance, I corrected this reappearance